contender-compose.yml

```diff
@@ -0,0 +1,48 @@
+setup:
```

Comment:
Would be cool if we didn't have to manually call setup. Feels like manually specifying rpc_urls for setup could get confusing in a large file (i.e. the user tries to spam on an alternate chain but forgets to run setup on that chain).

Instead, we could check if the scenario TOML file has any setup directives, then run `TestScenario::estimate_setup_cost` to find the required `min_balance`, then check the spam directives in the compose file to see which chains need to run which setup and run it automatically at the start.